live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/adapters/config/jackson/UpdateNodeEventSerializerTest.java

```diff
@@ -145,11 +150,15 @@ public void canCopeWithNullUsers()
     }
 
     @Test
-    public void shouldSetContentProperty()
+    public void shouldSetContentProperty() throws Exception
     {
+        String expectedDigest = EventUtils.generateSHA256Digest(NODE_ID, "cm:name", "1.0");
+
         UpdateNodeEvent event = new UpdateNodeEvent(NODE_ID, CREATE, SOURCE_ID, TIMESTAMP)
                 .addContentInstruction(new ContentProperty(CONTENT_PROPERTY, "content-id", "application/pdf",
-                        "application/msword", 123L, "something.doc"));
+                        "application/msword", 123L, "something.doc", expectedDigest));
+
+        event.setDigest(expectedDigest);
```

Comment:
I don't think this line is needed since it seems to be in the constructor above.